LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/inner-border-non-renderable.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="match" href="inner-border-ref.html">
+    <meta name=fuzzy content="0-144;0-484">
```

Comment:
can you include maxDifference=/totalPixels= for clarity?

Reply:
We don't need fuzziness anymore.